raster/r.kappa/prt_json.c

```diff
@@ -0,0 +1,94 @@
+#include <stdlib.h>
+#include <grass/gis.h>
+#include <grass/glocale.h>
+#include "kappa.h"
+#include "local_proto.h"
+
+
+void prn_json()
+{
+    bool first;
+    FILE *fd;
+
+    if (output != NULL)
+        fd = fopen(output, "w");
+    else
+        fd = stdout;
+
+    if (fd == NULL) {
+        G_fatal_error(_("Cannot open file <%s> to write JSON output"),
+                      output);
+        return;
```

Comment:
No need for `return;` (and thus no need for curly braces).

Reply:
You are totally right about return being unreachable. Although I am not a huge fan of no curly braces for a single statement as it is a potential trap in future.

Reply:
> Although I am not a huge fan of no curly braces for a single statement as it is a potential trap in future.

I agree with you on this, 100 per cent. I is currently GRASS "standard" though.